### DISCUSSION TITLE
iOS 18 support?

### DISCUSSION THREAD

**Discussion by mega6453** [doronz88/pymobiledevice3#1072] [Q&A] [answered]
There's a PR in a release states that fix support for iOS 18. So just wanted to know, pymobiledevice3 latest version will work with iOS 18 as well?
https://github.com/doronz88/pymobiledevice3/pull/1062

Thanks !

**Comment by doronz88** [comment, marked as answer]
It already supported iOS 18. The fix was specifically for restoring by a given IPSW

**Reply by mega6453** [reply]
ok Thanks
